crypto/err/err_prn.c

```diff
@@ -44,6 +45,117 @@ void ERR_print_errors_cb(int (*cb) (const char *str, size_t len, void *u),
     }
 }
 
+/* auxiliary function for incrementally reporting texts via the error queue */
+static void put_error(int lib, const char *func, int reason,
+                      const char *file, int line)
+{
+    ERR_new();
+    ERR_set_debug(file, line, func);
+    ERR_set_error(lib, reason, NULL /* no data here, so fmt is NULL */);
+}
+
+#define TYPICAL_MAX_OUTPUT_BEFORE_DATA 100
+#define MAX_DATA_LEN (ERR_PRINT_BUF_SIZE-TYPICAL_MAX_OUTPUT_BEFORE_DATA)
```

Comment:
Nit: spaces around "-"

Reply:
Oops, if we had run the tool from #10363 also on this source file we would have avoided this.

We'll fix this tomorrow, as well as two indentation nits that the tool found in addition.